packages/vscode-extension/package.json

```diff
@@ -71,7 +71,14 @@
 			{
 				"command": "suitecloud.uploadfile",
 				"key": "ctrl+alt+u",
-				"mac": "ctrl+alt+u"
+				"mac": "ctrl+alt+u",
+				"when": "resource =~ //FileCabinet/SuiteScripts/.*$/ || resource =~ //FileCabinet/Templates/.*$/ || resource =~ //FileCabinet/Templates/E-mail Templates/.*$/ || resource =~ //FileCabinet/Templates/Marketing Templates.*$/ || resource =~ //FileCabinet/Web Site Hosting Files/.*$/ || resource =~ //FileCabinet/SuiteScripts/.*$/ || resource =~ //FileCabinet/Templates/.*$/ || resource =~ //FileCabinet/Templates/E-mail Templates/.*$/ || resource =~ //FileCabinet/Templates/Marketing Templates.*$/ || resource =~ //FileCabinet/SuiteApps/.*$/"
```

Comment:
This could be much shorter if you use the "or" `|` operator inside the regex , or even if create a regex that at least share the common `/FileCabinet/`.